src/quo2/components/buttons/button/style.cljs

```diff
@@ -53,7 +53,7 @@
                       12))})
 
 (defn style-container
-  [{:keys [size disabled? border-radius background-color border-color icon-only? icon-top
+  [{:keys [size disabled border-radius background-color border-color icon-only? icon-top
```

Comment:
Why is question mark removed from `disabled?` prop?

Reply:
reverted.